ark/analysis/visualize_test.py

```diff
@@ -1,10 +1,54 @@
+import os
 import numpy as np
-from os import path
+import pytest
 
 from ark.analysis import visualize
 from ark.utils import test_utils
 
 
+def test_draw_boxplot():
+    # trim random data so we don't have to visualize as many facets
+    random_data = test_utils.make_segmented_csv(100)
+    random_data = random_data[random_data['PatientID'].isin(np.arange(1, 5))]
```

Comment:
we could make this an option in the test_utils if you think cases like this will become more common

Reply:
I only did it because I didn't want the boxplot to display like 10 different categories on one chart. I don't imagine we'll be doing this too often unless we plan to add a lot more faceted graphing functions in `visualize.py`, for example.